### PR TITLE
Moved news to more dropdown

### LINKED ISSUE #73: Move news to the dropdown menu
The `news` tab here on all the pages
image

news has become quite old and now it doesn't get updated. I think moving it to `more` dropdown will be better.

### PR DESCRIPTION
**Pull request by DineshKoushik**
### Description

- Changed the link reference position of new component to the `more` dropdown.

### Related issue

-  Closes #73

### CODE REVIEW

**Comment by DineshKoushik** [comment]
@Signior-X, please review this PR

File: index.html [hunk 1]
```diff
@@ -1,308 +1,420 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```
Comment (Signior-X) [inline index.html:3]: Dont change the whole formatting of the file. 
1. Keep the formatting same all over the project. Like we don't use any formatters as such. Just the default on in vscode. Better to turn off that as this is making change the whole pattern.
2. It is hard to see the changes you did. I also think, not only the index page, but other pages will also be needed this change.

**Review by Signior-X — CHANGES_REQUESTED** [review]

**Comment by DineshKoushik** [comment]
@Signior-X , Hey sorry for the use of formatter, I usually use it, so I happened to use it this time as well, I guess the new PR works. Please verify it.

**Comment by Signior-X** [comment]
Yes, it's perfect now!
Thanks for working on this. Merged :tada:

**Pull request merged**